sound/soc/intel/boards/sof_rt1308.c

```diff
@@ -236,6 +449,15 @@ static const struct platform_device_id board_ids[] = {
 					SOF_SSP_HDMI_CAPTURE_PRESENT |
 					SOF_RT1308_SPEAKER_AMP_PRESENT),
 	},
+	{
+		.name = "adl_cs35l41",
+		.driver_data = (kernel_ulong_t)(SOF_RT1308_SSP_CODEC(1) |
```

Comment:
you don't have an RT1308 on this platform.

Reply:
The macro SOF_RT1308_SSP_CODEC is used to encode the SSP port id for the amplifier. I rename it to SOF_AMPLIFIER_SSP.